fastchat/model/model_adapter.py

```diff
@@ -46,6 +48,11 @@ def load_model(self, model_path: str, from_pretrained_kwargs: dict):
         )
```

Comment:
1. Change L45 from 
```python
        tokenizer = AutoTokenizer.from_pretrained(model_path, use_fast=False)
```

to
```python
        tokenizer = AutoTokenizer.from_pretrained(model_path, use_fast=self.use_fast_tokenizer)
```

2. Then a lot of override functions can be removed, such as the `load_model` for OasstPythiaAdapter, StableLMAdapter, PhoenixAdapter